test/functional/services/test_subjects.js

```diff
@@ -37,6 +38,13 @@ export function TestSubjectsProvider({ getService }) {
       return await find.existsByDisplayedByCssSelector(testSubjSelector(selector), timeout);
     }
 
+    async existOrFail(selector, timeout = 1000) {
+      log.debug(`TestSubjects.existOrFail(${selector})`);
+      const doesExist = await this.exists(selector, timeout);
+      // Verify element exists, or else fail the test consuming this.
```

Comment:
nit: comment seems unnecessary.  But up to you!

Reply:
I normally would agree with you but I think the intention behind most of this test code becomes easily obscured so I've been erring on the side of caution with my comments.